webapp/src/ts/effects/contacts.effects.ts

```diff
@@ -60,16 +60,18 @@ export class ContactsEffects {
     return this.actions$.pipe(
       ofType(ContactActionList.setSelectedContact),
       withLatestFrom(
-        this.store.pipe(select(Selectors.getSelectedContact))
+        this.store.pipe(select(Selectors.getSelectedContact)),
+        this.store.pipe(select(Selectors.getUsersHomePlaceId)),
       ),
-      exhaustMap(([{ payload: { selected } }, previousSelectedContact]) => {
+      exhaustMap(([{ payload: { selected } }, previousSelectedContact, usersHomePlaceId]) => {
         if (!selected) {
           return []; // return an empty stream if there is no selected contact
         }
-        const refreshing = previousSelectedContact?.doc?._id === selected.id;
+        const refreshing = previousSelectedContact?.doc?._id === selected?.doc?._id ;
```

Comment:
Do we have a unit test on `contact-view-model-generator` that tests the structure of the `model` that it returns?

Reply:
No there is no test for the returned object.

Reply:
I think it's worth adding once, since this change exists.